packages/js/product-editor/src/blocks/toggle/block.json

```diff
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-toggle-field",
```

Comment:
Register this server side too in https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Admin/Features/ProductBlockEditor/BlockRegistry.php#L22